doc/install.rst

```diff
@@ -28,6 +28,51 @@ There are different ways to install scikit-learn:
 Installing the latest release
 =============================
 
+Quickstart
+----------
+
+.. raw:: html
+
+  <div class="install">
+       <p><strong>System</strong></p>
+          <input type="radio" name="os" id="quickstart-mac" checked>
+          <label for="quickstart-mac">macOS / OSX</label>
```

Comment:
I would remove the "/ OSX" part of the label. macOS is enough (and the OSX name does not exist anymore :). Let's keep things simple to improve readability.

Reply:
Done in b8f44c4023a0c5eb1d8a6107defed15bcf4d1166